### PR TITLE
More optimizations

### PR DESCRIPTION
**Pull request by parrish**
Mainly cutting down on redundant API calls on `/`, `/about/publications`, and `/collections`.

`/collections` could probably use some server-side improvements.

`/projects` now uses the `cards=true` param for consistency.

@camallen If we merge this, then we can just settle on that serializer and remove the other.

### CODE REVIEW

File: app/pages/about/publications-page.cjsx [hunk 1]
```diff
@@ -293,41 +306,43 @@ module.exports = React.createClass
...
     @setState currentSort: navItem
 
-  getAvatar: (project) ->
-    <PromiseRenderer promise={apiClient.type('projects').get(slug: project.slug).get('avatar')} pending={null}
-      catch={=> <img src="/assets/simple-pattern.jpg" />}
-      then={([avatar]) =>
-        <img src={avatar.src} alt="Project Avatar" />} />
+  avatarFor: (project) ->
+    src = if project?.avatar_src
+      "//#{ project.avatar_src }"
```
Comment (camallen) [inline app/pages/about/publications-page.cjsx]: Should the API return the `//` url prefix?

File: app/pages/home.cjsx [hunk 2]
```diff
@@ -42,17 +42,27 @@ counterpart.registerTranslations 'en',
...
+    src = if project.avatar_src
+      "//#{ project.avatar_src }"
+    else
+      '/assets/simple-avatar.jpg'
+    Promise.resolve src
+
   render: ->
     <div className="featured-projects">
-      <PromiseRenderer promise={apiClient.type('projects').get(FEATURED_PRODUCT_IDS)}>{(projects) =>
+      <PromiseRenderer promise={apiClient.type('projects').get(id: FEATURED_PRODUCT_IDS, cards: true)}>{(projects) =>
```
Comment (camallen) [inline app/pages/home.cjsx]: will this use the index route? If it's a show route, i'll have to make the cards context expandable to index / show.

File: app/pages/about/publications-page.cjsx (hunk 1 above)
Comment (parrish) [inline app/pages/about/publications-page.cjsx]: Either way. It's probably better if it returned a URL with a protocol though

File: app/pages/home.cjsx (hunk 2 above)
Comment (parrish) [inline app/pages/home.cjsx]: Yeah, it uses the index action with an id filter.

**Comment by parrish** [comment]
@brian-c I can't really find any more easy optimizations.  This should be good to check out again.  Thanks!

**Comment by brian-c** [comment]
No avatars are showing up on the Publications page. Looks good otherwise.

**Pull request merged**